src/PresentationalComponents/ZeroState/AppZeroState.js

```diff
@@ -51,84 +45,44 @@ const NewAppZeroState = ({
-                            appName={app}
-                            customInstructions={customInstructions}
-                            customButton={customButton}
-                            customText={customText}
-                            customTitle={customTitle}
-                            appId={appId}
-                        />
-                        {customSection && customSection}
-                        <AppSection appName={app}/>
-                        <ZeroStateFooter appName={app} />
-                    </Fragment>
-                }
-            </IntlProvider>
-    );
-};
+    }, [axios, children, customFetchResults, hasSystems]);
 
-//We will slowly migrate away from this
-const OldAppZeroState = ({
-    app,
-    customInstructions,
-    customButton,
-    customText,
-    customTitle,
-    appId
-}) => {
+    //If there are children, act as a wrapper, otherwise a component
     return (
         <IntlProvider>
-            <React.Fragment>
-                <ZeroStateBanner
-                    appName={app}
-                    customInstructions={customInstructions}
-                    customButton={customButton}
-                    customText={customText}
-                    customTitle={customTitle}
-                    appId={appId}
-                />
-                <AppSection appName={app}/>
-                <ZeroStateFooter appName={app} />
-            </React.Fragment>
+            {(children && hasSystems) ? (
+
+                children
+            ) : (
+                <>
+                    <ZeroStateBanner
+                        appName={app}
+                        customInstructions={customInstructions}
+                        customButton={customButton}
+                        customText={customText}
+                        customTitle={customTitle}
+                        appId={appId}
+                    />
+                    {customSection && customSection}
+                    <AppSection appName={app} />
+                    <ZeroStateFooter appName={app} />
+                </>
+            )}
         </IntlProvider>
     );
 };
 
 export default AppZeroState;
 
-OldAppZeroState.propTypes = {
-    app: propTypes.string,
-    customInstructions: propTypes.any,
-    customButton: propTypes.any,
-    customText: propTypes.string,
-    customTitle: propTypes.string,
-    appId: propTypes.string,
+AppZeroState.propTypes = {
     children: propTypes.any,
-    fetchSystem: propTypes.bool
-};
-
-NewAppZeroState.propTypes = {
-    app: propTypes.string,
     customInstructions: propTypes.any,
     customButton: propTypes.any,
     customText: propTypes.string,
     customTitle: propTypes.string,
     appId: propTypes.string,
-    children: propTypes.any,
     customFetchResults: propTypes.bool,
-    customSection: propTypes.node
-};
-AppZeroState.propTypes = {
-    children: propTypes.any,
+    customSection: propTypes.node,
     app: propTypes.oneOf([
```

Comment:
We should be able to refactor this to build a list based upon the zero state consts file.

Reply:
Then we don't need to worry about updating this when we add another app definition. ... Which TBH, I'd still move to the apps themselves and the zero state really only provides the layout/structure of the zero state, with only default content if an app doesn't provide content for a specific section.

Reply:
I'd like to take more time on this suggestion and maybe implement it in another PR. Since its already all built out theres not much change that needs to happen there. With this we would have duplicated code from here and the individual app. the 'default' content should be the ONLY content an app has.

Reply:
This is meant more for the second part. I implemented the list from the const file.

Reply:
Im gonna write this down for the next dev session, im having a hard time seeing the pros this. If we just make this a layout structure thing, we'd make Changes in all 15 apps essentially moving text from here to the individual apps. Though its fairly simple PRs, it seems more work than its worth. I also dont see why that'd benefit us. 

It would still be 1 PR per change, I guess the only thing that would change is the location of where the PR is.

Reply:
+1 to not delaying this any further. 
I have been waiting for zerostate to stabilize in order to work on [this](https://issues.redhat.com/browse/HMS-3459). 
If you want to make a larger change, please merge this first and allow a phased out approach for what is described above.